<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberDialog</class>
 <widget class="QWidget" name="MemberDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mitglied Eingabe</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="name">
      <attribute name="title">
       <string>Name</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Allgemeine Daten</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="memberIdLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Mitgliedsnummer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="memberNameLabel">
            <property name="text">
             <string>&amp;Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="memberName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="firstNameLabel">
            <property name="text">
             <string>&amp;Vorname</string>
            </property>
            <property name="buddy">
             <cstring>firstName</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="firstName"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="nicknameLabel">
            <property name="text">
             <string>Nic&amp;kname</string>
            </property>
            <property name="buddy">
             <cstring>nickname</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="nickname"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="memberId">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>groupBox</zorder>
      <zorder>memberId</zorder>
      <zorder>memberNameLabel</zorder>
      <zorder>memberName</zorder>
      <zorder>firstNameLabel</zorder>
      <zorder>firstName</zorder>
      <zorder>nicknameLabel</zorder>
      <zorder>nickname</zorder>
     </widget>
     <widget class="QWidget" name="adress">
      <attribute name="title">
       <string>Adresse</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Adresse</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="streetLabel">
            <property name="text">
             <string>&amp;Straße</string>
            </property>
            <property name="buddy">
             <cstring>street</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="street"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="cityLabel">
            <property name="text">
             <string>St&amp;adt</string>
            </property>
            <property name="buddy">
             <cstring>city</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="city"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="zipcodeLabel">
            <property name="text">
             <string>&amp;Postleitzahl</string>
            </property>
            <property name="buddy">
             <cstring>zipcode</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="zipcode"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>groupBox_2</zorder>
      <zorder>streetLabel</zorder>
      <zorder>street</zorder>
      <zorder>cityLabel</zorder>
      <zorder>city</zorder>
      <zorder>zipcodeLabel</zorder>
      <zorder>zipcode</zorder>
     </widget>
     <widget class="QWidget" name="etc">
      <attribute name="title">
       <string>Sonstiges</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Sonstiges</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="emailLabel">
            <property name="text">
             <string>&amp;Email</string>
            </property>
            <property name="buddy">
             <cstring>email</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="email"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="entryDateLabel">
            <property name="text">
             <string>Eintritts &amp;Datum</string>
            </property>
            <property name="buddy">
             <cstring>entryDate</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="entryDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2009</year>
              <month>12</month>
              <day>8</day>
             </date>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="contribution">
      <attribute name="title">
       <string>Beitrag</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Beiträge</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="feeLabel">
            <property name="text">
             <string>&amp;Beitrag</string>
            </property>
            <property name="buddy">
             <cstring>fee</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="fee"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="donationLabel">
            <property name="text">
             <string>Spende</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="donation"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="conInfoLabel">
            <property name="text">
             <string>Info</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="contributionInfo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="feeButton">
         <property name="text">
          <string>Beiträge</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bank">
      <attribute name="title">
       <string>Bank</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Bank Verbindung</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="bankNameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="bankName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="accountLabel">
            <property name="text">
             <string>Konto Nr.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="account"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="codeLabel">
            <property name="text">
             <string>BLZ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="code"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="infoSite">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="infoLabel">
            <property name="text">
             <string>&amp;Info</string>
            </property>
            <property name="buddy">
             <cstring>info</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPlainTextEdit" name="info"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="saldoButton">
     <property name="text">
      <string>Saldo Anzeigen</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Discard|QDialogButtonBox::SaveAll</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>memberName</tabstop>
  <tabstop>firstName</tabstop>
  <tabstop>nickname</tabstop>
  <tabstop>email</tabstop>
  <tabstop>entryDate</tabstop>
  <tabstop>info</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
